src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -618,20 +629,18 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
     needsReview: true,
     args: {
       solids: {
-        inputType: 'selection',
-        selectionTypes: ['path'],
-        selectionFilter: ['object'],
+        ...objectsTypesAndFilters,
+        inputType: 'selectionMixed',
         // TODO: turn back to true once engine supports it, the codemod and KCL are ready
         // Issue link: https://github.com/KittyCAD/engine/issues/3435
         multiple: false,
         required: true,
         hidden: (context) => Boolean(context.argumentsToSubmit.nodeToEdit),
       },
       tools: {
-        clearSelectionFirst: true,
+        ...objectsTypesAndFilters,
         inputType: 'selection',
-        selectionTypes: ['path'],
```

Comment:
Not having `'sweep'` here was limiting the ability to select the loft for #7567. This works now